imdclient/tests/minimalReader.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import select
+import time
+from imdclient.IMDClient import IMDClient
+from .utils import parse_host_port
+
+logger = logging.getLogger("imdclient.IMDClient")
+
+
+class minimalReader:
+    """
+    Minimal reader for testing purposes
+
+    Parameters
+    ----------
+    filename : str
+        a string of the form "host:port" where host is the hostname
+        or IP address of the listening MD engine server and port
+        is the port number.
+    n_atoms : int (optional)
+        number of atoms in the system. defaults to number of atoms
+        in the topology. don't set this unless you know what you're doing.
+    kwargs : dict (optional)
+        keyword arguments passed to the constructed :class:`IMDClient`
+    """
+
+    def __init__(self, filename, n_atoms=None, **kwargs):
```

Comment:
It looks as if `n_atoms` is really mandatory and not optional so why not just make it
```suggestion
    def __init__(self, filename, n_atoms, **kwargs):
```